### PR TITLE
add: card component

### PR DESCRIPTION
**Pull request by shilpe26**
Made Card Component with features:-
🔹Cards with Badges
🔹Card with Dismiss
🔹Cards with text overlay
🔹Text only cards
🔹Horizontal Card
🔹Vertical Card

### CODE REVIEW

File: css/component_styles/card.css [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+.img_container2 {
+    position: relative;
+    box-sizing: border-box;
+    background-repeat: no-repeat;
+    background-position: 50%;
+    background-size: cover;
+    color: inherit;
+    cursor: pointer;
+}
+
```
Comment (Hnh013) [inline css/component_styles/card.css:252]: more properties can be added to image base class for image (img1 & img_container2 has 4 same properties)

**Comment by Hnh013** [comment]
please use a variable or utiliy calss if its feasible for 1 rem size and white color (#FFF)

**Comment by anshikaa12** [comment]
code looks good to merge!

File: css/component_styles/card.css [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+.buttons {
+    display: flex;
+    margin-top: 1rem;
+}
+
+.buttons button:nth-child(1) {
+    width: 100%;
+    height: 40px;
```
Comment (rohit8686) [inline css/component_styles/card.css:60]: looks good, but same properties are being repeated (like width, height, border-radius), you can seperate them into another class

**Review by rohit8686 — APPROVED** [review]

**Comment by shilpe26** [comment]
Done all the changes.

**Pull request merged**